### DISCUSSION TITLE
Linking Word Count/Cloud to content, not search

### DISCUSSION THREAD

**Discussion by nocean** [caronchen/obsidian-chartsview-plugin#63] [Q&A] [answered]
I'm trying to create a word cloud (or word count cloud) that looks at a parameter called "topics" which contains a list of links (internal paths). I'm able to display this, but it renders the raw text, so if the topic link is to a note on Mozart it renders it as `[[mozart.md|Mozart]]`

Is there a way to get chartsview to render it as a link? So the cloud item can be clicked and open the note?

**Comment by caronchen** [comment, marked as answer]
I suggest taking a look at [this link](https://github.com/caronchen/obsidian-chartsview-plugin/issues/44#issuecomment-1454778120), it may provide a solution for your issue.
